File: Programmers/Lv0/코드처리하기/Solution.java
package Lv0.코드처리하기;

class Solution {
    public String solution(String code) {
        StringBuilder sb = new StringBuilder();
        boolean mode = false;

        char[] codeArr = code.toCharArray();

        for (int i = 0; i < code.length(); i++) {
            if (Character.isAlphabetic(codeArr[i])) {
                if (!mode && i % 2 == 0) {
                    sb.append(codeArr[i]);
                } else if(mode && i % 2 == 1) {
                    sb.append(codeArr[i]);
                }
            } else {
                mode = !mode;
            }
        }

        String answer = sb.toString();

        return answer.equals("") ? "EMPTY" : answer;
    }
}

File: Programmers/Lv0/코드처리하기/solution.py
def solution(code):
    answer = ""
    mode = False

    for i in range(len(code)):
        if code[i].isalpha():
            if (not mode and i % 2 == 0):
                answer += code[i]
            elif (mode and i % 2 == 1):
                answer += code[i]
        else:
            mode = not mode

    return answer if answer != "" else "EMPTY"
